docs/quickstart.md

```diff
@@ -2,7 +2,9 @@
 
 The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
+For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. Backups and restores for an on-premises cluster work the same way as the example.
```

Comment:
I would change this sentence to say something like "for simplicity this example uses Minio (an S3 compatible object store) running within your cluster. THIS IS A BAD IDEA FOR A PRODUCTION SCENARIO - you don't want to be storing your backups within the very cluster that you're trying to backup"

But I don't think this is actually the best place to discuss on-prem vs. cloud-provider (see other comments in this review)